server/datastore/mysql/migrations/tables/20230926082152_AddWindowsMDMTables.go

```diff
@@ -0,0 +1,69 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20230926082152, Down_20230926082152)
+}
+
+func Up_20230926082152(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+CREATE TABLE windows_mdm_commands (
+    -- command uuid is fleet-generated, so that a command has a unique id as
+    -- soon as it is enqueued. It is unique per device, unlike macOS commands
+    -- that may target multiple hosts. Note that a UUID is 36 chars, but for
+    -- consistency with nano_mdm_commands we use 127 here.
+    command_uuid    VARCHAR(127) NOT NULL,
```

Comment:
I wonder if that's what we want or if that should mimic macOS and the PK should be `command_uuid, device_id`? It wasn't clear on the ticket what the PK would be.

Reply:
After starting work on https://github.com/fleetdm/fleet/issues/13595, it is clear that a command_uuid may be related to multiple device ids, so I'll update this PR to reflect that and make the PK the combination of command and device.